zcash_client_sqlite/src/lib.rs

```diff
@@ -150,9 +150,33 @@ impl<P: consensus::Parameters> WalletRead for WalletDb<P> {
         wallet::get_unified_full_viewing_keys(self)
     }
 
-    fn get_address(&self, account: AccountId) -> Result<Option<PaymentAddress>, Self::Error> {
-        #[allow(deprecated)]
-        wallet::get_address(self, account)
+    fn get_address(&self, account: AccountId) -> Result<Option<UnifiedAddress>, Self::Error> {
```

Comment:
That method is not crate-private, that's the problem. Its implementation now diverges from `WalletRead::get_address`.

Reply:
Given that the vast majority of every function in `zcash_client_sqlite::wallet` is both public and deprecated, it seemed to me like the split was purely historic. If you want this impl to also be there, then it will be in a separate crate-private method.

Reply:
So, the deprecation was just to make it possible to eventually remove it from the public API. But the fact that the methods were originally public is, in my mind, a *feature* because I think that one should also expose the internals used to implement higher-level functionality, in general.